src/Catalyst.Core.Lib.Tests/UnitTests/DAO/DaoTests.cs

```diff
@@ -71,20 +75,18 @@ public DaoTests()
                 new DeltaDfsHashBroadcastMapperInitialiser(),
                 new FavouriteDeltaBroadcastMapperInitialiser(),
                 new CoinbaseEntryMapperInitialiser(),
-                new PublicEntryMapperInitialiser(),
+                new PublicEntryMapperInitialiser(_hashProvider),
                 new ConfidentialEntryMapperInitialiser(),
                 new TransactionBroadcastMapperInitialiser(),
-                new ContractEntryMapperInitialiser(),
-                new SignatureMapperInitialiser(),
-                new BaseEntryMapperInitialiser(), 
+                new SignatureMapperInitialiser()
             };
 
             _mapperProvider = new MapperProvider(_initialisers);
         }
 
         // ReSharper disable once UnusedMember.Local
-        private TDao GetMapper<TDao>() where TDao : IMapperInitializer => _initialisers.OfType<TDao>().First();
-        
+        private TDao GetMapper<TDao>() where TDao : IMapperInitializer { return _initialisers.OfType<TDao>().First(); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GetMapper'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)